export/hgnc_symbol_map.csv

```diff
@@ -1000,7 +1002,7 @@ DNAI1,2954
 DNAI2,18744
 DNAI3,30711
 DNAI4,26252
-DNAI7,29599
+DNAI7,None

```

Comment:
So I solved this by updating the INDRA resources then rebuilding this file in famplex, but I didn't commit/PR the updated files in INDRA since I don't know what the update policy is